packages/zoe/src/zoe.js

```diff
@@ -609,6 +619,9 @@ const makeZoe = (additionalEndowments = {}) => {
        */
       getInstance: instanceTable.get,
 
+      /** Get a notifier (see @agoric/notify) for the offer. */
+      getOfferNotifier: offerHandle => offerTable.get(offerHandle).notifier,
```

Comment:
BUG SECURITY Because of the name conflict I mention above, this reveals the notifier record with contains the updater!

Reply:
I should have recognized that because of the destructuring I had to do in the test.